src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v1-monitoring-integration.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Amazon API Gateway monitoring integration
+title: Amazon API Gateway v1 monitoring integration
```

Comment:
I would change this to "Amazon API Gateway REST (v1) monitoring integration

Reply:
@glennthomas1, i made the suggested changes.